src/main/java/uk/gov/mca/beacons/api/controllers/AccountHolderController.java

```diff
@@ -90,6 +92,22 @@ public AccountHolderIdDTO getAccountHolderId(
     );
   }
 
+  @PatchMapping(value = "/{id}")
+  @ResponseStatus(HttpStatus.OK)
+  @PreAuthorize("hasAuthority('APPROLE_UPDATE_RECORDS')")
```

Comment:
Probably okay, but just wanted to raise whether you think this role should be used to allow a user to update an account holder.

The description we have set for the role in Azure is: 
![image](https://user-images.githubusercontent.com/76042279/125044029-79033600-e093-11eb-9eef-82254509d5a9.png)

So thinking it could be a little bit confusing that updating the account holder is tied to this role? What are your thoughts?  We could leave it as is or create a separate role?

Reply:
good point, lets pick it up with the gang this morning

Reply:
updated the role descriptions for our CRUD operations